SYCL/ESIMD/api/functional/value_conv.hpp

```diff
@@ -51,15 +52,20 @@ std::vector<SrcT> generate_ref_conv_data() {
 
   // TODO: Implement functions for obtain lowest and max values without UB
   // cases.
-  static const SrcT nan = value<SrcT>::nan();
-  static const SrcT inf = value<SrcT>::inf();
-  static const SrcT denorm = value_conv<SrcT, DstT>::denorm_min();
   static const SrcT positive = static_cast<SrcT>(126.75);
+  static const SrcT max = 10;
+  static const SrcT min = 2;
```

Comment:
```suggestion
  static const SrcT min = std::min(-max, 0); // Use zero for unsigned types
```

Reply:
I suggest do not change these values because this is a temporary values that will be replaced with tested values with avoiding UB

Reply:
I've done it in this commit: b834199d3328e35b0d9dec7b608463b309421f3c